components/popover/popover.jsx

```diff
@@ -143,10 +143,26 @@ class Popover extends React.Component {
 		 * A footer is an optional. Buttons are often placed here.
 		 */
 		footer: PropTypes.node,
+		/**
+		 * CSS classes to be added to the `footer` of the popover.
+		 */
+		footerClassName: PropTypes.oneOfType([
+			PropTypes.array,
+			PropTypes.object,
+			PropTypes.string,
+		]),
 		/**
 		 * Prevents the Popover from changing position based on the viewport/window. If set to true your popover can extend outside the viewport _and_ overflow outside of a scrolling parent. If this happens, you might want to consider making the popover contents scrollable to fit the menu on the screen. When enabled, `position` `absolute` is used.
 		 */
 		hasStaticAlignment: PropTypes.bool,
+		/**
+		 * Removes display:inline-block from the trigger.
+		 */
+		hasNoTriggerStyles: PropTypes.bool,
+		/**
+		 * Will show the nubbin pointing from the dialog to the reference element. Positioning and offsets will be handled.
```

Comment:
This is supposed to be removes nubbin?

Reply:
This removes a `display:inline-block` on the thing that opens the popover.